internal/pkg/loader/loader.go

```diff
@@ -103,6 +103,12 @@ func loadFiles(files []*pack.File) (*pack.Pack, error) {
 			// object templates and helpers.
 			p.TemplateFiles = append(p.TemplateFiles, f)
 
+		case strings.HasPrefix(f.Name, "templates/") &&
+			!strings.HasSuffix(f.Name, ".tpl"):
```

Comment:
Do you have an example of a use case for this feature? Would it be possible to just change the `case` above to something like this?

```diff
- strings.HasSuffix(f.Name, ".nomad.tpl") ||
+ strings.HasSuffix(f.Name, ".tpl") ||
```

Reply:
Here's how I understood #299: we want files other than `.tpl` that could be found inside `templates/` to be rendered. Does that make sense? I can't really think of a usecase, I'm still pretty new to pack.

Reply:
#299 says: 

> Pack writers can include non-job templates alongside jobspecs in the /templates directory of a pack

So I guess we assume these would also have `.tpl` extension?

Reply:
Yeah, I think I'm reading that the same way as @angrycub. The last acceptance criteria mentions that:

> Any file that ends in “.tpl” is rendered

I think we can even avoid making assumptions about file content based on extenstion? For example, we could try to parse files as job and, if that fails, then we know that the file is not a job. Maybe it's a volume? So try to parse it as a volume etc.

This goes back to the discussion of having better scoped entities in Nomad, but that's a different discussion 😄

Reply:
I think I'm still a bit confused. I changed the conditional in 99f3544, let me know if this is what we want.